### ISSUE TITLE
issue trying to run master

### ISSUE THREAD

**Issue by stefanpenner** [ef4/initial-render-perf#2]
```
➜  initial-render-perf git:(master)  ember selenium --script=scripts/render-many.js
version: 1.13.1
Built project successfully. Stored in "selenium-dist".
no such element: Unable to locate element: {"method":"css selector","selector":".result"}
  (Session info: chrome=45.0.2454.85)
  (Driver info: chromedriver=2.19.346063 (38b35413bd4a486d436a9749e090454bc9ff6708),platform=Mac OS X 10.10.4 x86_64)
NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":".result"}
  (Session info: chrome=45.0.2454.85)
  (Driver info: chromedriver=2.19.346063 (38b35413bd4a486d436a9749e090454bc9ff6708),platform=Mac OS X 10.10.4 x86_64)
    at new bot.Error (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/atoms/error.js:108:18)
    at Object.bot.response.checkResponse (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/atoms/response.js:109:9)
    at /Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/webdriver/webdriver.js:379:20
    at promise.ControlFlow.runInFrame_ (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:1857:20)
    at goog.defineClass.notify (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:2448:25)
    at promise.Promise.notify_ (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:564:12)
    at Array.forEach (native)
    at promise.Promise.notifyAll_ (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:553:15)
    at goog.async.run.processWorkQueue (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/async/run.js:125:21)
    at process._tickCallback (node.js:357:9)
From: Task: <anonymous>
    at pollCondition (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:1614:14)
    at eval (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:1610:7)
    at new promise.Promise (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:368:5)
    at eval (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:1608:12)
    at promise.ControlFlow.runInFrame_ (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:1857:20)
    at promise.ControlFlow.runEventLoop_ (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:1729:8)
    at eval (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:2043:12)
    at goog.async.run.processWorkQueue (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/async/run.js:125:21)
    at process._tickCallback (node.js:357:9)
From: Task: <anonymous wait>
    at promise.ControlFlow.wait (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/goog/../webdriver/promise.js:1606:15)
    at webdriver.WebDriver.wait (/Users/stefanpenner/src/initial-render-perf/node_modules/ember-cli-selenium/node_modules/selenium-webdriver/lib/webdriver/webdriver.js:716:21)
    at runManyTest (/Users/stefanpenner/src/initial-render-perf/scripts/render-many.js:21:17)
    at /Users/stefanpenner/src/initial-render-perf/scripts/render-many.js:57:22
    at lib$rsvp$$internal$$tryCatch (/Users/stefanpenner/src/initial-render-perf/node_modules/rsvp/dist/rsvp.js:493:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/stefanpenner/src/initial-render-perf/node_modules/rsvp/dist/rsvp.js:505:17)
    at lib$rsvp$$internal$$publish (/Users/stefanpenner/src/initial-render-perf/node_modules/rsvp/dist/rsvp.js:476:11)
    at lib$rsvp$asap$$flush (/Users/stefanpenner/src/initial-render-perf/node_modules/rsvp/dist/rsvp.js:1198:9)
    at doNTCallback0 (node.js:408:9)
    at process._tickCallback (node.js:337:13)
```

**Comment by rwjblue** [comment]
Checking into it now...

**Comment by rwjblue** [comment]
Will be fixed by #3, sorry about that.

**Comment by rwjblue** [comment]
You can fix locally via `npm install --save-dev ember-perf@0.0.12` until that is landed.
